views/includes/footer.ejs

```diff
@@ -1,18 +1,31 @@
-<footer class="footer custom-content">
-    <div class="content has-text-centered">
+<footer class="footer">
+  <div class="container">
```

Comment:
content is not centered anymore and this is not beautiful.

Maybe you should choose the right element in https://bulma.io/documentation/.